vision_layer/vision_tasks/src/vision_node.cpp

```diff
@@ -0,0 +1,36 @@
+#include "buoy.h"
+#include "torpedo.h"
+#include "line.h"
+#include "markerDropper.h"
+#include "ros/ros.h"
+
+int main(int argc, char *argv[]){
+    
+    ros::init(argc, argv, "vision_node");    
+    double input = atof(argv[1]);
+    std::cout << input << std::endl;
+    if(input == 1)
```

Comment:
Instead of taking the argument as an input, try using server/client service provided by ros.